src/schema/validator.js

```diff
@@ -21,8 +21,11 @@ const validator = ajv({
   jsonPointers: true,
   verbose: true,
   schemas,
+  schemaId: 'auto',
 });
 
+validator.addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'));
```

Comment:
Can we move this import to the top maybe and make it an ES6 import?

`import jsonSchemaDraft06 from 'ajv/lib/refs/json-schema-draft-06.json';`

This will make the build more predictable and require less workarounds in the long term.

Reply:
Ah sure, I just copied that from the ajv upgrade guide :)